### DISCUSSION TITLE
Change additional channel infos

### DISCUSSION THREAD

**Discussion by Kariton** [iptv-org/community#4] [Ideas]
Hey,

First of all:
Thanks for your time and effort to maintain that repository or more this entire "IPTV" project!
I just searched for different IPTV Playlists and this looks really organized and promising.

I also came across some other ("not SO") awesome repository's - which I don't want to blame here!
(primary this is a _better_ project because here is EPG easily available)

While searching for playlists I discovered one very clever solution to keep ""meta information"" in the channel title.
They look like this:
> Channels which are not in HD are marked with an Ⓢ.
> Channels which use GeoIP blocking are marked with a Ⓖ.
> Channels which are live Youtube channels are marked with a Ⓨ.

and maybe for your other info:
"Not 24/7": ⊖
~"Timeout": ⊕ or  ⓧ~
~"Offline: ⦻~

or something.

IMO this is a nice, smart - not to say short - way to let the user know what is going on.

TBH as a Jellyfin user i can't always read the hole "normal" channel title if there is a channel icon available.
So the benefit might wary.

Edit1: And how does 'Timeout' and 'Offline' work? Where do they get "dropped"? I cant see them. (just discovered that)

**Comment by what-a** [comment]
I call dibs on Ⓝ meaning "newscast only".  😁

Is it safe to assume that all IPTV players will properly display ALT codes?
https://altcodeunicode.com/alt-codes-circled-number-letter-symbols-enclosed-alphanumerics/

**Comment by loucasal** [comment]
I would agree that this is a very user-friendly improvement. If maintainers are in favour and there is no technical reason why this wouldn't work, we could come up with a comprehensive list of relevant metadata.

**Comment by freearhey** [comment]
Maybe I'm missing something, but replacing plain text with ciphertext doesn't seem very user-friendly to me.
